js/apps/admin-ui/public/locales/lt/authentication.json

```diff
@@ -18,9 +18,9 @@
     "none": "jokio"
   },
   "flow": {
-    "browserFlow": "Autentifikacijos seka",
-    "registrationFlow": "Registracijos seka",
-    "directGrantFlow": "Tiesioginių teisių seka"
+    "browser": "Autentifikacijos seka",
+    "registration": "Registracijos seka",
+    "direct grant": "Tiesioginių teisių seka"
```

Comment:
we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"